### PR TITLE
Create ephemeral staging environments

### PR DESCRIPTION
**Pull request by crwilcox**

### CODE REVIEW

File: .github/workflows/pr-close.yaml [hunk 1]
```diff
@@ -0,0 +1,36 @@
+name: pr_close
+
+on:
+  pull_request:
+    branches: [ main ]
+    types: [closed]
```
Comment (crwilcox) [inline .github/workflows/pr-close.yaml:6]: nit: can we be consistent with brace spacing?

File: .github/workflows/pr-close.yaml [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+      name: 'Set up Google Cloud SDK'
+      with:
+        project_id: 'kr-gcping'
+
+    # Configure Workload Identity Federation via a credentials file.
+    - id: 'auth'
+      name: 'Authenticate to Google Cloud using Workload Identity Federation'
+      uses: 'google-github-actions/auth@v0'
+      with:
+        workload_identity_provider: 'projects/793166746566/locations/global/workloadIdentityPools/github-pool/providers/my-provider'
```
Comment (crwilcox) [inline .github/workflows/pr-close.yaml:29]: Super to see this using WIF

File: web/package.json [hunk 3]
```diff
@@ -19,5 +19,9 @@
     "eslint-plugin-prettier": "4.0.0",
     "parcel": "^2.0.0",
     "prettier": "2.4.1"
+  },
+  "engines": {
+    "node": ">16.0"
```
Comment (crwilcox) [inline web/package.json:24]: this engine doesn't match the actions above which specify 14.

File: .github/workflows/pr-close.yaml (hunk 1 above)
Comment (kkr16) [reply .github/workflows/pr-close.yaml]: Fixed.

File: .github/workflows/pr-close.yaml (hunk 2 above)
Comment (kkr16) [reply .github/workflows/pr-close.yaml]: Yep! Also moved the vars to use secrets instead.

File: web/package.json (hunk 3 above)
Comment (kkr16) [reply web/package.json]: Fixed. Bumped to v16.

**Pull request closed without merging**